jekyll/_cci2/contexts.md

```diff
@@ -237,121 +237,110 @@ to set up your CircleCI command line interface._
 
 To create an environment variable using our CLI, perform the following steps:
 
-1. If you have not already done so, find the right context name that will contain the new environment variable. Execute this command in the CLI: \
+1. If you have not already done so, find the right context name that will contain the new environment variable. Execute this command in the CLI:
 `circleci context list <vcs-type> <org-name>`
-2. Store a new environment variable under that context. Execute this command in the CLI: \
-`circleci context store-secret <vcs-type> <org-name> <context-name> <secret name>`
+2. Store a new environment variable under that context. Execute this command in the CLI:
+`circleci context store-secret <vcs-type> <org-name> <context-name> <env-var-name>`
```

Comment:
We used `env-var-name` in multiple places, so it seemed best to be consistent even though the term "secret" is more general. Right now we only support env vars for context contents (i.e. files, etc. are not a feature), so "env var" is maybe the better to lean into for now.